src/main/kotlin/br/com/zup/op/events/exception/model/FieldValidationCallback.kt

```diff
@@ -0,0 +1,6 @@
+package br.com.zup.op.events.exception.model
```

Comment:
remove this package `exception.custom` and move this class to `infra.validation` package